src/main/resources/assets/computercraft/lua/rom/apis/parallel.lua

```diff
@@ -14,7 +13,40 @@ local function create( ... )
     return tCos
 end
 
-local function runUntilLimit( _routines, _limit )
+local external = {
+    __index = {
+        add = function(bundle, ...)
+            local set = create(1, ...)
+            local rfuncs = {}
+            for i = 1, #set do
+                bundle[#bundle+1] = set[i]
+                rfuncs[#rfuncs+1] = function()
+                    local del = #bundle
```

Comment:
I don't believe this will behave how you expect, should probably be moved up to an upvalue

Reply:
Although on the other hand that would invalidate other removal functions, perhaps use the function value as the identifier?

Reply:
This is one of the discussion points I have in the PR description.

I could use the function, but I'd have to store that as well as the coroutines, in order to find the value to remove later on. I'm not sure if that's what is desired.

Reply:
One thing worth considering is what happens if you remove a coroutine while the runner is executing - there's a risk that a coroutine will be skipped. The simple solution would be to copy the coroutine table each time we modify it (or at least remove from it), so the list currently being iterated doesn't change, but will be updated next event.